README.md

```diff
@@ -424,14 +424,23 @@ Configuration goes in `.gmrc`, which is a JSON file with the following keys:
   values) — graphile-migrate does not perform any escaping for you. The special
   value `!ENV` will tell graphile-migrate to load the setting from the
   environment variable with the same name.
+- `beforeReset` — optional list of actions to execute before creating the
+  database, can be used to run save before database will be dropped with reset.
 - `afterReset` — optional list of actions to execute after the database has been
   created but before the migrations run, useful to set default permissions,
   install extensions or install external schemas like `graphile-worker` that
   your migrations may depend on. See "Actions" below.
+- `beforeAllMigrations` — optional list of actions to execute before the first
+  migration will ran, can be used to save some information about database state
+  before running migrations, for example collect stats.
 - `afterAllMigrations` — optional list of actions to execute after all the
   migrations have ran, useful for performing a tasks like dumping the database
   or regenerating dependent data (GraphQL schema, type definitions, etc). See
   "Actions" below.
+- `beforeCurrent` — optional list of actions to execute before `current.sql` is
+  executed. Can be used to save data before running `current.sql` and load again
+  with `afterCurrent`. This way you don't have to add test data after each
+  `current.sql` is executed.
```

Comment:
```suggestion
- `beforeCurrent` — optional list of actions to execute before `current.sql` is
  executed.
```

Reply:
Particularly here if we put this in the description people will ask how to do it and I'm not sure I can come up with a generic way for this to work in all cases right now. I've therefore removed all the usage suggestions.